### ISSUE TITLE
Issue with color.ml

### ISSUE THREAD

**Issue by happycamper31** [alexozer/flitter#16]
Hi,
I managed to install opam & the dependencies but when I try 'dune build' I get following error:
```
File "src/colors.ml", line 5, characters 8-26:
Error: Unbound value Color.of_hexstring
Hint: Did you mean to_hexstring?
```
I played around with the code, but i have no clue about ocaml, so I stopped after some fruitless tries. I hope you can figure out by these lines what is going on.
Regards

**Comment by tonussi** [comment]
I'm trying to understand why of_hexstring doesn't work properly. Maybe the output of of_hexstring is not being used correctly.

```
val of_hexstring : string -> t option
Parse a hexadecimal color code. Handles short format like #rgb or long format #rrggbb. Short format #abc corresponds to long format #aabbcc.
```

**Comment by tonussi** [comment]
The Color lib have a set of tests that could help us understand why the bug
https://github.com/anuragsoni/color/blob/master/test/color_conversion.ml

**Comment by alexozer** [comment]
Seems to be fixed as of 666c5483bc93efa6d01e0b7a927461269f8e14de.
